configs/config.toml.SAMPLE

```diff
@@ -42,7 +42,7 @@
 	# balance below which the client will start reporting errors in logs.
 	# The value should be passed in [wei].
 	#
-	# BalanceAlertThreshold = 500000000000000000 # 0.5 ETH (default value)
+	# BalanceAlertThreshold = "0.5 ether" # 0.5 ether (default value)
```

Comment:
It's a little late but not too late since we haven't released this yet. Don't you think that setting value to `0.5 ETH` wouldn't be more natural than setting it to `0.5 ether`? Although the latter is also correct, the former is more widely used.

Reply:
`ether` is the correct name for the unit https://ethdocs.org/en/latest/ether.html#denominations. `ETH` is a currency name on the exchanges.

Our implementation is consistent with `web3js`: https://web3js.readthedocs.io/en/v1.2.11/web3-utils.html#towei

We shouldn't define `ETH` in the `unit` enum, as this is not a correct unit name.

Reply:
OK, makes sense!